tests/Functional/tests/table-grid-utf8/table-grid-utf8.html

```diff
@@ -8,11 +8,7 @@
     <tbody>
         <tr>
             <td>x</td>
-            <td>
-                é à
-                <br>
-                é
-            </td>
+            <td>é à é</td>
```

Comment:
are you sure about this? I believe the old solution was correct as this is how you make multi line content in tables? Did you test this in sphinx?

Reply:
Yes, I verified all changed test output with Sphinx. Seems like multi line tables are created using the line block syntax (https://stackoverflow.com/a/13540865/1149495), but I guess this magical syntax is not yet supported by this parser.